### PR TITLE
Added script on how to load historic materialised data

### PR DESCRIPTION
**Pull request by dpatchigolla**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: scripts/load_historic_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    "connector = ProfilesConnector(config=creds)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "material_registry = connector.run_query(\"select * from material_registry_4\")\n",
```
Comment (shekhar-rudder) [inline scripts/load_historic_data.ipynb:104]: Will this query work for all warehouses ? Missing schema and case sensitivity might create problems

File: scripts/load_historic_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "homedir = os.path.expanduser(\"~\") \n",
+    "with open(os.path.join(homedir, \".pb/siteconfig.yaml\"), \"r\") as f:\n",
+    "    creds = yaml.safe_load(f)[\"connections\"][\"default\"][\"outputs\"][\"dev\"]"
```
Comment (shekhar-rudder) [inline scripts/load_historic_data.ipynb:61]: We can add a comment to verify connection and env names or make it a placeholder

File: scripts/load_historic_data.ipynb [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+    "import datetime"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "homedir = os.path.expanduser(\"~\") \n",
```
Comment (shekhar-rudder) [inline scripts/load_historic_data.ipynb:59]: Do we need to worry about windows ?

File: scripts/load_historic_data.ipynb [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+       "4                         1.0  "
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "material_data = connector.run_query(f\"select * from {material_name}\")\n",
```
Comment (shekhar-rudder) [inline scripts/load_historic_data.ipynb:287]: Same here

File: scripts/load_historic_data.ipynb [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+    "\n",
+    "`> pb run --end_time <required_time_in_unix_timestamp>`\n",
+    "\n",
+    "This notebook outlines these following steps to fetch that past materialised data:\n",
+    "1. Connect to warehouse using ProfilesConnector. The config can be accessed from the siteconfig file without changing any format.\n",
+    "2. Load the material registry table, and filter only the successful runs.\n",
+    "3. From material registry table, apply following filters to get the model_hash and seq_no of the right materialisation\n",
+    "    - `model_type` = 'entity_traits_360'\n",
+    "    - `end_ts` = '<unix time from above, after converting to timestamp - see below>'\n",
+    "4. There may be multiple materialisations of same end ts. In such cases, it returns the most recently created materialisation details - seq_no and the model hash are the attributes that define a materialisation\n",
```
Comment (shekhar-rudder) [inline scripts/load_historic_data.ipynb:19]: I believe wht also creates a view for 360 table with a constant name. We can use this view to abstract away the seq_no complexity

Comment (dpatchigolla) [reply scripts/load_historic_data.ipynb:19]: This is required for fetching historic data. That view without the seq_no points to the most recent end_ts, so this is required.

File: scripts/load_historic_data.ipynb (hunk 2 above)
Comment (dpatchigolla) [reply scripts/load_historic_data.ipynb]: Added it as a variable along with all other variables

File: scripts/load_historic_data.ipynb (hunk 1 above)
Comment (dpatchigolla) [reply scripts/load_historic_data.ipynb]: Good point. it wouldn't work on redshift. The prospect is currently on redshift, and made it redshift compatible. Did not test on BigQuery for now. This is more of illustrative, so it's okay if it didn't work on BigQuery.

File: scripts/load_historic_data.ipynb (hunk 3 above)
Comment (dpatchigolla) [reply scripts/load_historic_data.ipynb:63]: Addressed. Made it windows compatible.

**Review by shekhar-rudder — APPROVED** [review]

File: scripts/load_historic_data.ipynb (hunk 2 above)
Comment (shekhar-rudder) [reply scripts/load_historic_data.ipynb]: What about env name ?

**Pull request merged**